src/main/resources/view/MainWindow.fxml

```diff
@@ -13,7 +13,7 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.stage.Stage?>
 
-<fx:root maxHeight="800.0" maxWidth="1200.0" minHeight="700.0" minWidth="800.0" onCloseRequest="#handleExit" title="AddressBook π" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root minHeight="600.0" minWidth="800.0" onCloseRequest="#handleExit" title="AddressBook π" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
There is a warning about javaFX version 17, could you change it to version 11 for all the fxml files 👍 

![image](https://user-images.githubusercontent.com/77260849/162112537-7eff1c03-2056-463e-9851-09d24da89ae1.png)

Reply:
Changed!